### PR TITLE
refactor: Show network connectivity error tooltip

### PR DESCRIPTION
**Pull request by pawanpaudel93**
## Summary

This PR adds the network connectivity tooltip for AO tokens when the balance fetch is failed.

### CODE REVIEW

File: src/components/popup/Token.tsx [hunk 1]
```diff
@@ -228,6 +232,17 @@ const DegradedMessage: React.ReactNode = (
   </div>
 );
 
+const NetworkErrorMessage: React.ReactNode = (
+  <div style={{ textAlign: "center" }}>
+    <div style={{ fontSize: "14px" }}>Network issue detected.</div>
+    <div style={{ fontSize: "12px", color: "#a3a3a3" }}>
+      Your ISP or VPN is blocking access to AO.
```
Comment (7i7o) [inline src/components/popup/Token.tsx]: Let's i18n these two messages

**Review by 7i7o — CHANGES_REQUESTED** [review]

File: src/components/popup/Token.tsx (hunk 1 above)
Comment (pawanpaudel93) [reply src/components/popup/Token.tsx]: Sure, I will do that.

**Comment by pawanpaudel93** [comment]
@7i7o since we cannot get the same fetch error status as we see in the browser, we don't know if it was failed due to ISP/VPN blocking ao. I did add a common fetch error check that occurs on if fetch is failed but do you have idea on how we can make it better ?

**Comment by 7i7o** [comment]
> @7i7o since we cannot get the same fetch error status as we see in the browser, we don't know if it was failed due to ISP/VPN blocking ao. I did add a common fetch error check that occurs on if fetch is failed but do you have idea on how we can make it better ?

Actually those are the best error messages we have from the users that experienced the problem

**Review by 7i7o — APPROVED** [review]
tACK 👍

**Pull request merged**